src/Support/SelectFields.php

```diff
@@ -207,7 +207,12 @@ protected static function handleFields(array $queryArgs, array $requestedFields,
 
                         $with[$relationsKey] = self::getSelectableFieldsAndRelations($queryArgs, $field, $newParentType, $customQuery, false);
                     } else {
-                        self::handleFields($queryArgs, $field, $fieldObject->config['type'], $select, $with);
+                        if (method_exists($fieldObject->getType(), 'getWrappedType')) {
```

Comment:
Why would you check for the method and not if its an instance of `WrappingType`? See existing use for this in the very same class.

I think in this case I would prefer a ternary operator, something like:
```php
$type = … instanceof …
    ? …
    : …;
```